ChangeLog

```diff
@@ -1,3 +1,7 @@
+2023-05-26 Walter Somerville <waltersom@gmail.com>
```

Comment:
The old emacs generated format has two spaces here between date and name, and name and email.  Can you adjust?

Thanks for thinking of the ChangeLog, I appreciate it!

Reply:
Aagh, whitespace, always trips me up!

Done.